sop/modules/OSD/pages/CephNodeDown.adoc

```diff
@@ -1,44 +1,85 @@
-// begin header
-ifdef::env-github[]
-:tip-caption: :bulb:
-:note-caption: :information_source:
-:important-caption: :heavy_exclamation_mark:
-:caution-caption: :fire:
-:warning-caption: :warning:
-endif::[]
-:numbered:
-:toc: macro
-:toc-title: pass:[<b>Table of Contents</b>]
-// end header
-= CephNodeDown
+include::partial$alert_header.adoc[]
+
+:alertname: CephNodeDown
+:message: A storage node went down. Please check the node immediately. The alert should contain the node name.
+:description: A storage node went down. Please check the node immediately. The alert should contain the node name.
+:detailed_description:
+:resolution_description: Resolution to this alert involves replacing the failed node with a new node of the same or larger capacity and ensuring OCS osd pods are properly placed on the new node.
+:severity: Error
+:ceph-component: rook-ceph-mgr
+= {alertname}
 
 toc::[]
 
 == Check Description 
 
-*Severity:* Error
+*Severity:* {severity}
 
 *Potential Customer Impact:* High
 
 
 == Overview
+{description}
 
-A storage node went down. Please check the node immediately. The alert should contain the node name.
-
-== Prerequsities
+{detailed_description}
 
+== Prerequisites
 include::partial$prerequisites.adoc[]
 
 ==  Alert
-include::partial$gather_logs.adoc[]
 
+=== Make changes to solve alert
 
+.Document the current OCS pods (running and failing):
+[source,role="execute"]
+----
+oc -n openshift-storage get pods
+----
 
+WIP: Link to node replacement SOP
```

Comment:
It may be helpful to troubleshoot why the node is down:

// The expected number of each machine set.
oc get machinesets -n openshift-machine-api

// Describe these machines
oc get machines -n openshift-machine-api